src/hooks/__tests__/use-fetch.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { useFetch } from '../use-fetch'
+import { renderHook, waitFor } from '@testing-library/react'
+
+describe('useFetch', () => {
+    it('should fetch data immmediately', async () => {
+        const url = 'https://jsonplaceholder.typicode.com/posts'
+
+        const { result } = renderHook(() => useFetch(url))
+
+        expect(result.current.data).toBe(undefined)
+
+        expect(result.current.error).toBe(undefined)
+
+        await waitFor(() => expect(result.current.data).not.toBeUndefined())
+
+        await waitFor(() => expect(result.current.error).toBe(undefined))
+
+        await waitFor(() => expect(result.current.isLoading).toBe(false))
+    })
+
+    it('should not fetch data immmediately trigger is false', async () => {
+        const url = 'https://jsonplaceholder.typicode.com/posts'
+
+        const { result } = renderHook(() => useFetch(url, false))
+
+        expect(result.current.data).toBe(undefined)
+
+        expect(result.current.error).toBe(undefined)
+    })
+
+    it('should fetch data immediately trigger is true', async () => {
```

Comment:
This test fails in codespace:

![image](https://github.com/alamenai/rehook/assets/20143684/89ca3d3c-70c5-472f-895e-21433eb4ee66)

Reply:
They are well now:

![image](https://github.com/alamenai/rehook/assets/20143684/687ba089-3fd4-42d6-876e-eca683c87b79)

Reply:
Resolved with this [commit](https://github.com/alamenai/rehook/pull/43/commits/54e5a32bfd149e6e196f9e31aee66bf475a7de34).